_content/tour/rus/functions.article

```diff
@@ -110,73 +109,72 @@ arguments to a function. To define a variadic parameter, you use an ellipsis
        return sum
    }
 
-You can call this function with any number of integers.
+Вы можете вызвать эту функцию с любым количеством целых чисел.
 
-** Anonymous Functions
+** Анонимные Функции
 
-Go supports anonymous functions, also known as closures. These are functions
-without a name, and they can be assigned to variables and used as arguments to
-other functions. Closures are often used in Go for tasks like defining inline
-functions or for concurrent programming using goroutines.
+Go поддерживает анонимные функции, также известные как замыкания. Это функции
+без имени, и их можно присваивать переменным и использовать в качестве аргументов
+других функций. Замыкания часто используются в Go для таких задач как определение
+встроенных функций или для конкурентного программирования с использованием горутин.
 
    add := func(x, y int) int {
        return x + y
    }
 
    result := add(3, 5)
 
-** Function as a Type
+** Функция как Тип
 
-In Go, functions can also be used as types. This allows you to define functions
-that take other functions as arguments or return them as results. It's a powerful
-feature for implementing higher-order functions and callbacks.
+В Go функции также могут использоваться как типы. Это позволяет определять функции,
+которые принимают другие функции в качестве аргументов или возвращают их как результат.
+Это мощный механизм для реализации функций высокого порядка и колбэков.
 
    type MathFunc func(int, int) int
 
    func operate(x, y int, op MathFunc) int {
        return op(x, y)
    }
 
-You can then pass functions as arguments to `operate`.
+Затем вы можете передать функции в качестве аргументов `operate`.
 
-** Defer and Panic
+** Отложенные Вызовы и Паника
 
-Go provides two special built-in functions called `defer` and `panic` for handling
-exceptional situations and resource management. `defer` is used to schedule a function
-call to be executed just before the function returns, while `panic` is used to
-trigger a run-time error and unwind the stack.
+Go предоставляет две специальные встроенные функции, называемые `defer` и `panic`, для обработки
+ошибок и управления ресурсами. `defer` используется для планирования вызова функции
```

Comment:
```suggestion
исключителньых ситуаций и управления ресурсами. `defer` используется для планирования вызова функции
```

In Go, we differentiate "error" and "exceptional situation". Errors are values of `error` type, and panics show that something truly exceptional happened. This is quite an important moment for understanding of Go, that's why I leave this comment.